test/blocks/text/text.test.js

```diff
@@ -31,4 +31,33 @@ describe('text block', () => {
       expect(body).to.exist;
     });
   });
+
+  describe('Link Farm', () => {
+    it('should check if the "link-farm" class is present', () => {
+      const element = document.querySelector('.link-farm');
+      expect(element).to.exist;
+    });
+  
+    it('should check if the "h3" elements are added when necessary', () => {
+      const element = document.querySelector('.link-farm');
+      const foregroundDiv = element.querySelectorAll('.foreground')[1];
+      const headingElem = foregroundDiv.querySelectorAll('h3');
+      expect(headingElem.length).to.equal(4);
+    });
+    it('should add no-heading id to the h3 element', () => {
+      const element = document.querySelector('.link-farm');
+      const foregroundDiv = element.querySelectorAll('.foreground')[1];
+      const headingElem = foregroundDiv.querySelector('#no-heading');
```

Comment:
```suggestion
      const headingElem = foregroundDiv.querySelector('.no-heading');
```

Reply:
Could likely also make this selector one line `const headingElem = document.querySelectorAll('.link-farm .foreground .no-heading')`